src/components/MyList/MyList.tsx

```diff
@@ -0,0 +1,34 @@
+import { memo } from 'react';
+import { User } from '../../models/user';
+import { MyListItem } from '../MyListItem/MyListItem';
+import './MyList.css';
+
+interface MyListProps {
+  users: User[];
+}
+
+function MyList(props: MyListProps) {
+  const { users } = props;
+
+  console.log('render list');
```

Comment:
Always remember, careful with the details when opening a PR

Reply:
removed